jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/CommonDirectory.java

```diff
@@ -349,13 +349,18 @@ static private boolean isCommonControlFile(String dirMemberName) {
 	 */
 	static void deleteStaleDirectories(String myId) {
 		long myUid = IPC.getUid();
+		if (LOGGING_DISABLED != loggingStatus) {
+			IPC.logMessage("deleting stale dirs, myId = " + myId + ", myUid = " + myUid); //$NON-NLS-1$ //$NON-NLS-2$
+		}
+
```

Comment:
nitpick: please remove this empty line.

Reply:
Adding this empty line before main block is my initial intention
I can remove it.